examples/src/main/scala/org/platanios/tensorflow/examples/python2scala/LinearRegressionFromRestoredPythonModel.scala

```diff
@@ -0,0 +1,123 @@
+package org.platanios.tensorflow.examples.python2scala
+
+import java.io._
+import java.nio.file.Paths
+
+import com.typesafe.scalalogging.Logger
+import org.platanios.tensorflow.api.core.Graph
+import org.platanios.tensorflow.api.ops.Output
+import org.platanios.tensorflow.api.{Op, Session, Shape, Tensor, tf}
+import org.slf4j.LoggerFactory
+import org.tensorflow.framework.MetaGraphDef
+
+import scala.collection.mutable.ArrayBuffer
+import scala.util.Random
+
+/**
+  * @author Luca Tagliabue
+  *         Purpose: take a model trained with the Python API of Tensor Flow and restore
```

Comment:
You should move the description in the doctoring, above the author tag and make it unindented. Also, you don't need to specify the versions used as I plan to maintain this and it should be working when I make updates. :)